MiniApp/Classes/MiniApp.swift

```diff
@@ -71,6 +71,14 @@ public class MiniApp: NSObject {
         return []
     }
 
+    /// Gets the list of downloaded Mini apps info and associated custom permissions status
+    /// - Returns:Dictionary of MiniAppInfo and respective custom permissions info
```

Comment:
Is this return format convenient to use on iOS? Basically `MiniAppInfo` is the key and `MASDKCustomPermissionModel` is the value, right?

From the Android perspective, this would be a little difficult to use. On Android, we would probably use `Pair` which allows two values to be tied together in a single object. Like this: `Array<Pair<MiniAppInfo, MASDKCustomPermissionModel>>` and then the resulting usage looks something like this:

```
val pairs = Array(
  miniAppInfo1 to permissions1,
  miniAppInfo2 to permissions2,
 // ...
)

pairs.forEach {
  val info = it.first
  val permissions = it.second
  // use data...
}
```

Does iOS have something like `Pair`? Or maybe we could create a custom model for this like `MiniAppInfoPermissionsPair`? What do you think?

Reply:
Yes, in iOS we have [Tuple](https://docs.swift.org/swift-book/ReferenceManual/Types.html). I guess we can go with Tuple/Pair approach instead of custom model. Looks much more cleaner.

Reply:
Done